### PR TITLE
Contributors: add alum

### PR DESCRIPTION
**Pull request by bitschmidty**
Opened as an alternative to https://github.com/bitcoinops/bitcoinops.github.io/pull/741

Proposed here: https://github.com/bitcoinops/bitcoinops.github.io/pull/741#issuecomment-1084773391

Preview of associated "about" page: https://deploy-preview-761--bitcoinops.netlify.app/about/

### CODE REVIEW

**Comment by moneyball** [comment]
Looked at the preview here. LGTM.

**Comment by bitschmidty** [comment]
Added a commit for James and Marcin as alumni

**Comment by harding** [comment]
LGTM.  I think maybe we could rename the "Optech Contributors" section to "Current Contributors" and this new section to "Previous Contributors", but it doesn't really matter to me.

**Comment by jonatack** [comment]
ACK, tested the preview on Firefox and it renders fine for me.

File: about.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ MIT license. Anyone is welcome to contribute by opening issues and pull
 requests, reviewing newsletters and other material, and contributing
 translations. Our most regular contributors are:
 
-{% include contributors.html %}
+{% assign contributors = site.data.contributors.contributors | sort: "name" %}
+{% include contributors.html id="contributors" %}
 
 {% include sponsors.html %}
+
+## Optech Alum
```
Comment (Xekyo) [inline about.md:50]:
```suggestion
## Former Optech Contributors
```

**Review by Xekyo — COMMENTED** [review]
I'm glad that this proposal seems to gain some support.

To jump on the bandwagon with the bikeshedding of section titles, I'd like to posit "Optech Contributors" and "Former Optech Contributors". But any of the above work fine for me, and I wouldn't mind if we wrapped it up. :)

**Comment by jnewbery** [comment]
Thanks @bitschmidty. Please add me to the "former contributors" section.

**Review by Xekyo — APPROVED** [review]
Built locally, looked over the former contributor section, checked links.

LGTM

**Comment by jnewbery** [comment]
ACK. Thanks @bitschmidty

**Comment by jamesob** [comment]
So happy for what optech has become! Keep up the great work, everyone.

**Pull request merged**